style.css

```diff
@@ -1 +1,285 @@
-/*  Add your styling here */
+:root {
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+}
+
+nav {
+    display: none;
+}
+
+/* HEADER   */
+
+.header-top-section {
+    background-color: rgb(247, 247, 247);
+    display: flex;
+    justify-content: space-between;
+    height: 150px;
+    padding: 20px 20px 10px 20px;
+    gap: 100px;
+}
+
+.header-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    align-content: space-between;
+    justify-content: space-between;
+}
+
+.logo {
+    width: 80px;
+    height: 80px;
+}
+
+h1{ 
+    font-size: 0.75em;
+    font-weight: 300;
+    text-align: right;
+    /* padding: 0 30px 30px 0; */
+}
+
+/* MAIN */
+
+main {
+    margin: 20px;
+    /* overflow: hidden; */
+}
+
+/* WELCOME */
+
+.welcome-section,.welcome-text {
+    margin-bottom: 30px;
+}
+
+.welcome-img {
+    width: 99.9%;
+}
+
+/* CARDS */
+
+.cards {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 20px;
+    width: 100%;
+}
+
+.card-text {
+    display: none;
+}
+
+.card-img {
+    height: 100%;
+    /* margin-left: auto;
+    margin-right: auto; */
+    width: 100%;
+}
+
+
+/* FOOTER */
+
+footer {
+    background-color: pink;
+    height: 75px;
+    padding: 15px 0 15px 15px;;
+}
+
+.footer-links {
+    padding-bottom: 15px;;
+}
+
+.footer-links-text{
+    font-size: 0.6em;
+    margin: 2px;
+    color: rgb(97, 97, 97);
+    text-decoration: none;
+}
+
+.fab{
+    color:  rgb(97, 97, 97);
+}
+
+@media only screen and (min-width: 540px) {
```

Comment:
Well done doing mobile first and including multiple, appropriate breakpoints for small, medium and large screens